meshtastic/node_runtime/transport_runtime.py

```
@@ -0,0 +1,350 @@
+    """Delete-channel rewrite execution plan."""
+
+    pre_delete_admin_index: int
+    post_delete_admin_index: int
+    switch_after_admin_slot_rewrite: bool
+    channels_to_rewrite: list[channel_pb2.Channel]
+
+
+class _NodeDeleteChannelRuntime:
+    """Owns delete-channel validation, planning, and ordered rewrite execution."""
+
+    def __init__(self, node: "Node", *, channel_write_runtime: _NodeChannelWriteRuntime):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)